input/MergeStars/particles.dat

```diff
@@ -0,0 +1,1000 @@
+2.000000000000000179e+33 1.451866231686910400e+17 1.560847637893675200e+17 1.787318274063598720e+17 2.161734950536991656e+08 1.795544997827437818e+08 1.034577834384659231e+08
```

Comment:
I don't either maybe ask Matthew? Why do you need a particular conda setup?

Reply:
I thought it best for purposes of reproducibility, and for for ease-of-use, if users could recreate an identical environment and then simply execute "run.sh" and produce all the plots. I suppose that the only dependency ics.py has is numpy, so it might be possibly just to check whether the user has numpy installed in their environment.

Reply:
Thanks for the help @mabruzzo , I'll have a go at that tomorrow. And thanks for spotting that I wasn't actually setting a seed in ics.py.

Reply:
Yes thanks @mabruzzo . I agree that we should avoid having to install conda. I would say though that assuming that yt is installed for the tests is very reasonable. I'm not sure but the circle-ci tests may already pull YT down when running the tests. If not I think that is something that could be added without much difficulty.

Reply:
I can confirm that the Circle-Ci test does already pull `yt` to run the tests. (I added that myself a while back for the VL+CT tests).